src/RSCommand.cc

```diff
@@ -145,6 +151,14 @@ void RSCommand::add_link(PMGDQuery& query, const Json::Value& link,
 
 AddEntity::AddEntity() : RSCommand("AddEntity")
 {
+    _storage_blob = VDMSConfig::instance()
```

Comment:
Shouldn't this variable be called _blob_path? This isn't a blob by itself.

Reply:
this is following the same convention on images. where the path to pngs/jpg etc is _storage_format